app/kafka/producer.js

```diff
@@ -0,0 +1,20 @@
+const { Kafka } = require('kafkajs');
+
+const kafka = new Kafka({
+	clientId: 'graphql-schema-registry-server',
+	brokers: ['gql-schema-registry-kafka:9092'],
+});
+
+const producer = kafka.producer();
+const run = async () => {
+	// Producing
+	await producer.connect();
+	await producer.send({
+		topic: 'test-topic',
+		messages: [{ value: 'Hello KafkaJS user!' }],
```

Comment:
I think the schema of this payload should match format individual service update like its in our rest API.. for example `GET /schema/latest` gives me
```
    {
      "id": 4,
      "service_id": 5,
      "version": "v1",
      "name": "service_a",
      "url": "http://localhost:6101",
      "added_time": "2021-10-01T10:16:12.000Z",
      "type_defs": "\n\ttype Query {\n\t\thello: String\n\t}\n",
      "is_active": 1
    },
```